### PR TITLE
Adding sofc flowsheet

### PR DESCRIPTION
**Pull request by AlexNoring**
## Proposed changes:
- This PR adds the sofc model from the SOFC-IES project

### Legal Acknowledgement

By contributing to this software project, I agree to the following terms and conditions for my contribution:

1. I agree my contributions are submitted under the license terms described in the LICENSE.txt file at the top level of this directory.
2. I represent I am authorized to make the contributions and grant the license. If my employer has rights to intellectual property that includes these contributions, I represent that I have received permission to make contributions and grant the required license on behalf of that employer.

### CODE REVIEW

File: notebook_index.yml
Comment (andrewlee94) [inline notebook_index.yml:163]: Typo: capital U in Fuel

File: src/Examples/Flowsheets/power_generation/sofc/sofc.ipynb
Comment (andrewlee94) [inline src/Examples/Flowsheets/power_generation/sofc/sofc.ipynb:32]: These deprecation warnings probably need to be addressed - we don;t want these showing up in examples if possible.

File: src/Examples/Flowsheets/power_generation/sofc/sofc.ipynb
Comment (andrewlee94) [inline src/Examples/Flowsheets/power_generation/sofc/sofc.ipynb:184]: Can we address or suppress this output (or clear the output)? With all this output here, it is a lot of scrolling to read the notebook.

File: src/Examples/Flowsheets/power_generation/sofc/sofc_rom.py
Comment (andrewlee94) [inline src/Examples/Flowsheets/power_generation/sofc/sofc_rom.py:13]: It would be nice to have some documentation here to explain what this does and how the data is used. At the moment it is not clear what all of this is doing.

**Review by andrewlee94 — CHANGES_REQUESTED** [review]
There is a lot to review here, but it would be nice if we could clean up the output in the notebook file. I also noted one typo in what I could follow.

**Comment by andrewlee94** [comment]
@AlexNoring FYI, we are transitioning to use a new repository for the examples, and we are not merging any more PRs in `examples-pse`.

The new repo can be found here: https://github.com/IDAES/examples
The README has some instructions on adding new examples.

**Comment by eslickj** [comment]
> @AlexNoring FYI, we are transitioning to use a new repository for the examples, and we are not merging any more PRs in `examples-pse`.
> 
> The new repo can be found here: https://github.com/IDAES/examples The README has some instructions on adding new examples.

I think now we have until Jan 16, so I think we can merge this when it's ready.

File: src/Examples/Flowsheets/power_generation/sofc/sofc.ipynb
Comment (eslickj) [reply src/Examples/Flowsheets/power_generation/sofc/sofc.ipynb]: I think the way to do this is to set the logger level to ERROR.  At least for this one cell.  Then you won't see warnings.

**Review by eslickj — APPROVED** [review]

**Comment by AlexNoring** [comment]
The PFD looks fine outside of Github

**Pull request merged**
